online_test/settings.py

```diff
@@ -122,20 +122,25 @@
 
 STATIC_ROOT = 'yaksh/static/'
 
+
+
+
 # Set this varable to <True> if smtp-server is not allowing to send email.
-EMAIL_USE_TLS = False
+EMAIL_USE_TLS = True
+
+EMAIL_HOST = 'smtp.gmail.com'
 
-EMAIL_HOST = 'your_email_host'
+EMAIL_PORT = '587'
 
-EMAIL_PORT = 'your_email_port'
+EMAIL_HOST_USER = 'yakshcertificate@gmail.com'
 
-EMAIL_HOST_USER = 'email_host_user'
+EMAIL_HOST_PASSWORD = 'elite2019'
 
-EMAIL_HOST_PASSWORD = 'email_host_password'
```

Comment:
Please remove  the Host user and Host password. Username and Passwords should never be released in public. Let this be as it is.

Reply:
Shall we remove email_port and email_host